src/client/sandbox/node/window.js

```diff
@@ -724,6 +727,56 @@ export default class WindowSandbox extends SandboxBase {
         this._overrideUrlPropDescriptor('protocol', nativeMethods.anchorProtocolGetter, nativeMethods.anchorProtocolSetter);
         this._overrideUrlPropDescriptor('search', nativeMethods.anchorSearchGetter, nativeMethods.anchorSearchSetter);
 
+        overrideDescriptor(window.SVGImageElement.prototype, 'href', {
+            getter: function () {
+                const imageHref = nativeMethods.imageHrefGetter.call(this);
+
+                return Object.defineProperty(imageHref, CONTEXT_IMAGE_ELEMENT, { value: this });
```

Comment:
I think we need to add `this[CONTEXT_IMAGE_ELEMENT]` condition. Also add `configurable: true` to descriptor.
And use the `defineProperty` function from `nativeMethods`.